assessmentModel/src/commonMain/kotlin/survey/FormStepState.kt

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.assessmentmodel.survey
+
+import org.sagebionetworks.assessmentmodel.*
+import org.sagebionetworks.assessmentmodel.navigation.BranchNodeState
+import org.sagebionetworks.assessmentmodel.navigation.LeafNodeState
+import org.sagebionetworks.assessmentmodel.navigation.previousResult
+
+interface FormStepState : LeafNodeState {
```

Comment:
LeafNodeState  - a leaf node is intended to be used by a step fragment and step view model.